packages/cli/src/ceramic-daemon.ts

```diff
@@ -1,33 +1,35 @@
+import * as fs from 'fs'
 import express, { Request, Response } from 'express'
 import type { CeramicConfig } from '@ceramicnetwork/core'
-import Ceramic from '@ceramicnetwork/core'
+import { Ceramic } from '@ceramicnetwork/core'
 import { RotatingFileStream } from '@ceramicnetwork/logger'
-import { buildIpfsConnection } from './build-ipfs-connection.util'
-import { S3StateStore } from './s3-state-store'
+import { buildIpfsConnection } from './build-ipfs-connection.util.js'
+import { S3StateStore } from './s3-state-store.js'
 import {
   DiagnosticsLogger,
   LoggerProvider,
   MultiQuery,
   StreamUtils,
   SyncOptions,
 } from '@ceramicnetwork/common'
-import StreamID, { StreamType } from '@ceramicnetwork/streamid'
-import ThreeIdResolver from '@ceramicnetwork/3id-did-resolver'
-import KeyDidResolver from 'key-did-resolver'
-import PkhDidResolver from 'pkh-did-resolver'
+import { StreamID, StreamType } from '@ceramicnetwork/streamid'
+import * as ThreeIdResolver from '@ceramicnetwork/3id-did-resolver'
+import * as KeyDidResolver from 'key-did-resolver'
+import * as PkhDidResolver from 'pkh-did-resolver'
 import EthrDidResolver from 'ethr-did-resolver'
-import NftDidResolver from 'nft-did-resolver'
-import SafeDidResolver from 'safe-did-resolver'
+import * as NftDidResolver from 'nft-did-resolver'
+// import SafeDidResolver from 'safe-did-resolver'
```

Comment:
Somethings up with safe DID?

Reply:
It was not released. Thanks for catching that. Restored it back.